### PR TITLE
test: refactor upgradetest

### PR DESCRIPTION
**Pull request by arirubinstein**
refs: #7499

further improvements

### CODE REVIEW

File: packages/deployment/upgrade-test/Makefile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+REPOSITORY = agoric/upgrade-test
+
+build:
+	docker build --progress=plain -t $(REPOSITORY):latest -f Dockerfile upgrade-test-scripts
+
+run:
+	docker run -it -e "DEST=1" -v "$$(PWD):/workspace" --entrypoint bash $(REPOSITORY):latest
```
Comment (raphdev) [inline packages/deployment/upgrade-test/Makefile]: This breaks for me in linux, while it works fine in mac:

```sh
make run
docker run -it -e "DEST=1" -v "$(PWD):/workspace" --entrypoint bash agoric/upgrade-test:latest
/bin/sh: 1: PWD: not found
root@d58ce43f96fe:/usr/src/agoric-sdk# ls /workspace
ls: cannot access '/workspace': No such file or directory
```

```sh
make run
docker run -it -e "DEST=1" -v "$(PWD):/workspace" --entrypoint bash agoric/upgrade-test:latest
root@75308157bf15:/usr/src/agoric-sdk#
```

Looks like when it resolves it's not portable across shells:

```sh
echo $SHELL; echo $(PWD)
/bin/bash
bash: PWD: command not found
```
```sh
echo $SHELL; echo $(PWD)
/bin/zsh
/Users/raphs/Code/agoric-sdk/packages/deployment/upgrade-test
```

Perhaps we should use:
```suggestion
	docker run -it -e "DEST=1" -v "$${PWD}:/workspace" --entrypoint bash $(REPOSITORY):latest
```

File: packages/deployment/upgrade-test/Readme.md [hunk 2]
```diff
@@ -2,12 +2,8 @@
...
-This will run the upgradehandler for the latest upgrade
-
-2. `docker run -it -v "$PWD:/workspace" --env "SLOGFILE=/workspace/slog.slog" --entrypoint bash agoric/upgrade-test:latest -c "agd start --home netstate"`
-
-## To test with local sdk changes
 1. `(cd ../ && make docker-build-sdk)`
-2. then follow steps 1 and 2 above
+2. `make build`
+3. `make run`
+4. `tmux`
```
Comment (raphdev) [inline packages/deployment/upgrade-test/Readme.md:8]: Note: `tmux -CC` also works with terminals that support integration (e.g. iTerm2). Maybe worth mentioning?

```suggestion
4. `tmux` or `tmux -CC`
```

**Review by raphdev — APPROVED** [review]
Confirmed the new instructions work. Fixed minor issue with shell compat.

**Comment by arirubinstein** [comment]
I give up

**Pull request merged**